src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -28,11 +49,38 @@ const colorModeIcon: Record<ColorScheme, ReactChild> = {
   system: <ColorModeSystemIcon />,
 };
 
+interface KeyboardShortcutProps {
+  keybinding: string[];
+  description: string;
+}
+
+const KeyboardShortcut = ({
+  keybinding,
+  description,
+}: KeyboardShortcutProps) => {
+  const shortcutSegments = keybinding.map((segment) => (
+    <kbd className={styles.kbd} key={`${keybinding}-${segment}`}>
+      {segment}
+    </kbd>
+  ));
+
+  return (
+    <div className={styles.keyboardShortcutRow}>
+      <Text>{description}</Text>
+      <Text size="large">
+        <Inline space="xxsmall">{shortcutSegments}</Inline>
+      </Text>
+    </div>
+  );
+};
+
 interface SettingsPanelProps {}
 
 export default ({}: SettingsPanelProps) => {
   const [{ editorPosition, colorScheme }, dispatch] = useContext(StoreContext);
 
+  const keybindings = useMemo(getKeyBindings, [navigator.platform]);
```

Comment:
Since this is a pure component we can memo it entirely. The contents doesn't change, but it will rerender if context changes.

```suggestion
export default React.memo(() => {
  const [{ editorPosition, colorScheme }, dispatch] = useContext(StoreContext);

  const keybindings = getKeybindings();
```